ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -683,87 +683,6 @@ tests:
         OPENSHIFT_VERSION=4.14
       PLATFORM: vsphere
     workflow: assisted-vsphere
-- as: e2e-vsphere-assisted-periodic
```

Comment:
don't we have vsphere/nutanix periodics in 2.8/2.9?

Reply:
Yes my bad thanks